packages/create-block/lib/prompts.js

```diff
@@ -73,11 +73,39 @@ const category = {
 	choices: [ 'common', 'embed', 'formatting', 'layout', 'widgets' ],
 };
 
+const author = {
+	type: 'input',
+	name: 'author',
+	message:
+		"The author name — this should be a list of wordpress.org userid's:",
+};
+
+const license = {
+	type: 'input',
+	name: 'license',
+	message: 'The plugin license:',
+	filter( input ) {
```

Comment:
Rather than using the `filter` method, it's possible to leave it to default values that are defined here:

https://github.com/WordPress/gutenberg/blob/099929ba87d61403b5b0c30571f58c70954b387e/packages/create-block/lib/templates.js#L15-L58